docker-compose.yml

```diff
@@ -110,7 +111,7 @@ services:
 
   web:
     image: warehouse:docker-compose
-    command: gunicorn --reload --reload-extra-file=warehouse/api/openapi.yaml -b 0.0.0.0:8000 --access-logfile - --error-logfile - warehouse.wsgi:application
+    command: gunicorn --reload --reload-extra-file=warehouse/api/openapi.yaml -w 4 -b 0.0.0.0:8000 --access-logfile - --error-logfile - warehouse.wsgi:application
```

Comment:
This change disabled the debug toolbar, as shown:

```
/opt/warehouse/lib/python3.12/site-packages/pyramid_debugtoolbar/toolbar.py:244: UserWarning: pyramid_debugtoolbar has detected that the application is being served by a forking / multiprocess web server. The toolbar relies on global state to work and is not compatible with this environment. The toolbar will be disabled.
```

What was the behavior you observed that led to adding this?

Reply:
oops!

I experienced lockups pretty regularly when I had two browsers open with different profiles attempting to interact with the web server.